src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const pricePerDay = 40;
+
```

Comment:
Create a const standartPrice = days * pricePerDay;

Reply:
I couldnt reopen this pull request. I am sending a link to a new one with my fixes => https://github.com/mate-academy/js_task-transportation-on-vacation/pull/3369